app/actions/createStrategy.ts

```diff
@@ -0,0 +1,17 @@
+'use server'
+
+import { PrismaClient } from "@prisma/client"
+
+export async function createStrategy(id: string, name: string, text: string, sql: string) {
```

Comment:
UI is progressive (doesn't show anything related to strategies right now since backend is not deployed) and this is create only, so it is "safe" in that sense for now. This will need auth as we deploy the Electric backend!